homeassistant/components/motion_blinds/cover.py

```diff
@@ -160,6 +163,13 @@ def name(self):
     @property
     def available(self):
         """Return True if entity is available."""
+        if not self._blind.available:
+            track_point_in_time(
```

Comment:
It looks like this may schedule multiple refresh requests, one for any time this property is accessed.